UD-Viz-Core/src/Utils/DataViewer/js/BaseDemo.js

```diff
@@ -0,0 +1,1053 @@
+import { ModuleView } from '../../ModuleView/ModuleView.js';
```

Comment:
What is the main difference between your baseDemo.js and the original one ? 
If I get it right, you added some variable corresponding to some  wms or wfs layer. Since our demo are always based on Lyon, I think that we could create an interface with all WFS / WMS sources that you described which allow the viewer to chose which one to load in his demo.
Perhaps we can talk about that with EBO, in order to find the best solution to do that.

Reply:
There is no longer a baseDemo specific to this demo. The orginal baseDemo was modified a bit, and new layers are currently added in the demo.js file

Reply:
UD-Viz should have no (or almost) reference to specific data.
All what is specific should end up in demos that use UD-VIZ:
the standard place to collect such (docker based) demos is in [UD-Reproducibility/Demos/](https://github.com/VCityTeam/UD-Reproducibility/tree/master/Demos/ValleeChimie).
Now it could still be that we need a tiny 3DTiles to illustrate some
UD-Viz features but my guess is that UD-Viz itself (as a software component)
should not end-up being a catalog for Lyon related data...